Appraisals

```diff
@@ -40,10 +40,19 @@ appraise 'spree-4-2' do
   gem 'rspec-rails',              '~> 4.0.0.beta2'
 end
 
-appraise 'spree-master' do
-  gem 'spree_core',               github: 'spree/spree',                  branch: 'master'
-  gem 'spree_backend',            github: 'spree/spree',                  branch: 'master'
-  gem 'spree_auth_devise',        github: 'spree/spree_auth_devise',      branch: 'master'
+appraise 'spree-4-3' do
+  gem 'spree_core',               '~> 4.3.0'
+  gem 'spree_backend',            '~> 4.3.0'
+  gem 'spree_auth_devise',        '~> 4.3.0'
+  gem 'rails-controller-testing'
+  gem 'sass-rails'
+  gem 'rspec-rails',              '~> 4.0.0.beta2'
+end
+
+appraise 'spree-main' do
+  gem 'spree_core',               '~> 4.3'
+  gem 'spree_backend',            '~> 4.3'
+  gem 'spree_auth_devise',        '~> 4.4', '>= 4.4.1'
```

Comment:
@damianlegawiec There is some problem with these gems, CircleCi fails if we lead it to specific branch:
https://app.circleci.com/pipelines/github/spree-contrib/spree_avatax_official/52/workflows/991215cd-4bc5-4ec8-a677-70b0faf1bed2/jobs/764

https://app.circleci.com/pipelines/github/spree-contrib/spree_avatax_official/53/workflows/934a697b-a24b-4de6-991e-8ef9090a1403/jobs/782

https://app.circleci.com/pipelines/github/spree-contrib/spree_avatax_official/54/workflows/0ee92528-3292-4f36-b284-0805a329f676/jobs/790

but if we set up versions there is another problem: https://app.circleci.com/pipelines/github/spree-contrib/spree_avatax_official/56/workflows/3c68ba09-096e-45c0-9af8-bc420a88f06e/jobs/818


IMO, for spree-main we should not set specific versions, it should take it from branch, I guess.
I consulted with Damian, and we both do not know what's wrong with it and how to fix it.

Could you please take a look at this? :pray:

Reply:
@agnieszkajacek spree main is spree 4.4, not 4.3

Reply:
Yeah, but I set dependencies for spree-main, and 4.3 is sets for them not for spree main 
Not really know what do you mean @damianlegawiec

Reply:
But it does not work at all if I set the main branch, CircleCI does not pass

Reply:
you need to point it to github - `gem 'spree_auth_deivse', github: 'spree/spree_auth_devise', branch: 'main'`

Reply:
maybe I didn't say what I wanted right. From the beginning, I had it set up in the same way as you wrote above. The problem is that when we point to a specific branch, CircleCI shows the errors I mentioned above.

I reverted changes and we point to github and the branch https://github.com/spree-contrib/spree_avatax_official/pull/171/commits/e152d058c3eed937289cb805f6f20fd33f39a528
and as you can see, CircleCI fails:
![image](https://user-images.githubusercontent.com/13260586/143895352-57e21dbc-f614-442c-ae0c-4bfbbd4b881e.png)